layer.md

```diff
@@ -156,3 +156,13 @@ Note that this opaque file will apply to _all_ children, including sub-directori
 Implementations SHOULD generate layers using _explicit whiteout_ files, but MUST accept both.
 
 Any given image is likely to be composed of several of these Image Filesystem Changeset tar archives.
+
+# Foreign Layers
+
+Certain layers, due to legal requirements, may not be regularly distributable.
+Typically, one can download such layers but they should never be uploaded.
```

Comment:
On Wed, Sep 07, 2016 at 11:43:09AM -0700, Stephen Day wrote:
> > @@ -156,3 +156,13 @@ Note that this opaque file will apply to _all_ children, including sub-directori
> >  Implementations SHOULD generate layers using _explicit whiteout_ files, but MUST accept both.
> >  
> >  Any given image is likely to be composed of several of these Image Filesystem Changeset tar archives.
> > +
> > +# Foreign Layers
> > +
> > +Certain layers, due to legal requirements, may not be regularly distributable.
> > +Typically, one can download such layers but they should never be uploaded.
>
> This is an important clarification and frames the stronger language
> below.

What is this line adding beyond the earlier:

  Certain layers, due to legal requirements, may not be regularly distributable.

and the planned [1]:

  It is implementation defined whether or not implementations
  distribute layers tagged with this media type.

Can you pin down the information the other lines are missing that
you're trying to convey here?

[1]: https://github.com/opencontainers/image-spec/pull/233#issuecomment-245354663